ReleaseProcessAutomation.IntegrationTests/Git/CommandLineGitClientTests.cs

```diff
@@ -186,7 +186,7 @@ public void GetAncestors_DevelopBehindReleaseBranch_ReturnsOne ()
   }
 
   [Test]
-  public void GetAncestors_ReleaseBehindDevelopBranch_ReturnsOne ()
+  public void GetAncestors_WithTwoExpectedAncestors_ReturnsBothNameMatched ()
```

Comment:
`ReturnsNameMatched` is there a better way to say this, e.g. `ReturnsMatchingName` or `ReturnsNameBasedOnAncestor` etc?